### PR TITLE
Delete unnecessary packages

### LINKED ISSUE #239: Consider deleting some packages
We have a lot of packages, some of them don't provide added value, so I would consider deleting them.

| Simulation                   | Proposed Action | Comment                                                                                                                                                                     |
|------------------------------|-----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| webots_ros2_abb              | delete          | No added value, just a dummy simulation                                                                                                                                     |
| webots_ros2_demos            | rename          | It demonstrates how to use multiple robots. Alternatively, the multi-robot simulations can be demonstrated with UR5 robots (already exists in webots_ros2_universal_robot)  |
| webots_ros2_core             | deprecate       | After the webots_ros2_driver is mature the webots_ros2_core should be deprecated                                                                                            |
| webots_ros2_examples         | delete          | No added value                                                                                                                                                              |
| webots_ros2_tiago            | delete          | No added value                                                                                                                                                              |
| webots_ros2_tutorials        | move            | It should be maintained by SoftIllusion                                                                                                                                     |
| webots_ros2_universal_robot  | cleanup        | Very useful, but there are simulations inside that don't provide added value (should be deleted)                                                                            |
| webots_ros2_epuck            | move            | Maybe move to epuck_ros2                                                                                                                                                    |
| webots_ros2_ur_e_description | use upstream    | Hopefully, the ur_description will be published soon for ROS 2, so we can use the upstream version                                                                          |

### PR DESCRIPTION
**Pull request by lukicdarkoo**
Resolves #239

Deleted packages:
- webots_ros2_examples
- webots_ros2_tutorials

Replaced packages:
- webots_ros2_tiago (this one integrates Navigation2)

### CODE REVIEW

**Review by BenjaminHug8 — CHANGES_REQUESTED** [review]
It works nicely on my side!

Beside waiting for the new CI to finished and my question for the `webots_ros2/package.xml`, I just have some warnings most probably due to the deletion of packages that I think we can resolve:
`[0.392s] WARNING:colcon.colcon_ros.prefix_path.ament:The path '/home/benjamin/ros2_ws/install/webots_ros2_ur_e_description' in the environment variable AMENT_PREFIX_PATH doesn't exist

[0.392s] WARNING:colcon.colcon_ros.prefix_path.ament:The path '/home/benjamin/ros2_ws/install/webots_ros2_demos' in the environment variable AMENT_PREFIX_PATH doesn't exist

[0.392s] WARNING:colcon.colcon_ros.prefix_path.ament:The path '/home/benjamin/ros2_ws/install/webots_ros2_tutorials' in the environment variable AMENT_PREFIX_PATH doesn't exist

[0.392s] WARNING:colcon.colcon_ros.prefix_path.ament:The path '/home/benjamin/ros2_ws/install/webots_ros2_examples' in the environment variable AMENT_PREFIX_PATH doesn't exist

[0.392s] WARNING:colcon.colcon_ros.prefix_path.ament:The path '/home/benjamin/ros2_ws/install/webots_ros2_abb' in the environment variable AMENT_PREFIX_PATH doesn't exist
`

File: webots_ros2/package.xml [hunk 1]
```diff
@@ -18,14 +18,11 @@
   <exec_depend>webots_ros2_core</exec_depend>
   <exec_depend>webots_ros2_driver</exec_depend>
   <exec_depend>webots_ros2_epuck</exec_depend>
-  <exec_depend>webots_ros2_examples</exec_depend>
   <exec_depend>webots_ros2_importer</exec_depend>
   <exec_depend>webots_ros2_mavic</exec_depend>
   <exec_depend>webots_ros2_msgs</exec_depend>
   <exec_depend>webots_ros2_tesla</exec_depend>
-  <exec_depend>webots_ros2_tiago</exec_depend>
```
[the remark this answers is not in the document]
Comment (lukicdarkoo) [reply webots_ros2/package.xml:26]: Yes, you right

**Comment by lukicdarkoo** [comment]
> I just have some warnings most probably due to the deletion of packages that I think we can resolve

`rm -rf build install log` and building the workspace in a new terminal should make the warnings disappear.

**Review by BenjaminHug8 — APPROVED** [review]
Perfect, thank you !

**Pull request merged**